postman/Manage Subscriber.postman_collection.json

```diff
@@ -0,0 +1,281 @@
+						"   pm.expect(pm.environment.has(\"consumer_secret\")).to.eql(true) && pm.expect(pm.environment.get(\"consumer_secret\")).not.eql('');",
+						"});",
+						"",
+						"pm.test(\"Check access_token is defined and not empty\", function(){",
+						"   pm.expect(pm.environment.has(\"access_token\")).to.eql(true) && pm.expect(pm.environment.get(\"access_token\")).not.eql('');",
+						"});",
+						"",
+						"pm.test(\"Check token_secret is defined and not empty\", function(){",
+						"   pm.expect(pm.environment.has(\"token_secret\")).to.eql(true) && pm.expect(pm.environment.get(\"token_secret\")).not.eql('');",
+						"});",
+						"",
+						"pm.test(\"Successfully get an account Id\", function(){",
+						"    pm.response.to.have.status(200);",
+						"    var jsonData = pm.response.json();",
+						"    pm.expect(jsonData.entries.length).to.be.above(0);",
+						"    pm.globals.set(\"accountId\",jsonData.entries[0].id ) ;   ",
+						"});",
+						"",
+						"pm.test(\"Successfully submitted request\", function() {",
+						"    pm.response.to.have.status(200);",
+						"});",
+						"",
+						""
+					],
+					"type": "text/javascript"
+				}
+			}],
+			"request": {
+				"method": "GET",
+				"header": [],
+				"body": {
+					"mode": "raw",
+					"raw": ""
+				},
+				"url": {
+					"raw": "{{base_url}}/accounts",
+					"host": [
+						"{{base_url}}"
+					],
+					"path": [
+						"accounts"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "Get Lists",
+			"event": [{
+				"listen": "test",
+				"script": {
+					"id": "f6163948-14a0-48e5-ad18-ee37ab1a3c0d",
+					"exec": [
+						"pm.test(\"Successfully get a  random list\", function() {",
+						"    pm.response.to.have.status(200);",
+						"});",
+						"",
+						"pm.test(\"At least two lists found\", function() {",
+						"    var jsonData = pm.response.json();",
+						"    pm.expect(jsonData.total_size).to.be.above(1);",
+						"    pm.globals.set(\"listId\",jsonData.entries[0].id );",
+						"});"
+					],
+					"type": "text/javascript"
+				}
+			}],
+			"request": {
+				"method": "GET",
+				"header": [],
+				"body": {
+					"mode": "raw",
+					"raw": ""
+				},
+				"url": {
+					"raw": "{{base_url}}/accounts/{{accountId}}/lists",
+					"host": [
+						"{{base_url}}"
+					],
+					"path": [
+						"accounts",
+						"{{accountId}}",
+						"lists"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "Check is subscriber is already on list",
+			"event": [{
+				"listen": "test",
+				"script": {
+					"id": "8e652d7e-fd45-4591-a567-b0c9589d9ce6",
+					"exec": [
+						"pm.test(\"Successfully seaarched list for subscriber\", function(){",
+						"    pm.response.to.have.status(200);",
+						"    pm.test(\"Subscriber email is not empty or null\", function() {",
+						"        pm.expect(pm.environment.has(\"subscriber_email\")).to.eql(true) && pm.expect(pm.environment.get(\"subscriber_email\")).not.eql(\"\");",
```

Comment:
This will always fail, see previous comment about comparing a string to a boolean using `eql`.